packages/data-point/lib/entity-types/entity-hash/factory.js

```diff
@@ -93,10 +93,16 @@ function validateCompose (entityId, compose, validKeys) {
   })
 }
 
-function create (spec) {
+/**
+ * Creates new Entity Object
+ * @param  {Object} spec - spec
+ * @param {string} id - Entity id
+ * @return {Object} Entity Object
```

Comment:
EntityHash instead of Object would match the naming for the other create functions